simulations/index.html

```diff
@@ -66,7 +66,7 @@ <h5 class="sideheader">Charts</h5>
 <div class="container top-bar top-buffer">
   <div class="row small-buffer">
     <div class="col l2 m12 s12">
-      <h5 class="sideheader">Simulations</h5>
+      <h5 class="sideheader" id="simulations">Simulations</h5>
```

Comment:
id tag makes this linkable elsewhere on the site, right? And the charts tag is not yet used?